config/templates/metasploit-framework-wrappers/msfwrapper.erb

```diff
@@ -37,7 +37,7 @@ check_db() {
 }
 
 check_path() {
-  if [ "`which msfconsole`" = "" ]; then
+  if [ ! hash $cmd 2>/dev/null ]; then
```

Comment:
"if" isn't the same as "if [ !"

Not sure if this really makes the function any more readable, but I can rearrange it.